webgpu.h

```diff
@@ -346,7 +346,7 @@ typedef enum WGPUSType {
     WGPUSType_Invalid = 0x00000000,
     WGPUSType_SurfaceDescriptorFromMetalLayer = 0x00000001,
     WGPUSType_SurfaceDescriptorFromWindowsHWND = 0x00000002,
-    WGPUSType_SurfaceDescriptorFromXlibWindow = 0x00000003,
+    WGPUSType_SurfaceDescriptorFromXlib = 0x00000003,
```

Comment:
I think in [#140](https://github.com/webgpu-native/webgpu-headers/pull/140#discussion_r778264107) it was discussed to rename `Xlib`. So I think it's okay to keep this `XlibWindow` ?

Reply:
Oops I actually changed dawn to fit naming in webgpu-headers just for this. Let me change things back